src/components/Tabs/__tests__/TabsItem.test.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import {render, screen, fireEvent} from '@testing-library/react';
+import {TabsItem} from '../TabsItem';
+import {GearIcon} from '../../icons/GearIcon';
+
+const tabId = 'tab-id';
+const tabTitle = 'Tab title';
+
+describe.only('TabsItem', () => {
```

Comment:
only

Reply:
there is actually no need to wrap all tests in common `describe`, jest is adding test name anyway

```
PASS src/components/Tabs/__tests__/TabsItem.test.tsx (14.139 s)
  TabsItem
    ✓ should render tab item by default (29 ms)
```